builtin/settingtypes.txt

```diff
@@ -624,26 +624,6 @@ debanding (Enable Debanding) bool true
 #    Requires: shaders, enable_post_processing
```

Comment:
The `[**Bloom]` header (some lines above, where github doesn't allow me to comment) should probably be removed.